Bottom: 0b81645b8fa73c08bd9680a5de5682b731b52903
Top:    0033577f95c2433d1ac71ba52a3ed399ba29ee04
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-20 23:18:10 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
